src/components/molecules/DatePickerInput/classes.module.scss

```diff
@@ -56,10 +56,6 @@
 .datePickerWrapper {
   width: 125px;
   position: relative;
-
-  &.increasedZIndex {
```

Comment:
Indeed I forgot to remove the use of this, I removed this class because in the modal all the inputs were on top of the opened date picker calendars. Overall everything looks good in modals without this class